cx_Freeze/macdist.py

```diff
@@ -324,6 +324,10 @@ def run(self):
                 self.copy_tree(resource, self.resourcesDir + '/' +
                                destination)
             else:
+                parent_dirs = os.path.dirname(self.resourcesDir + '/' +
+                                              destination)
+                if not os.path.exists(parent_dirs):
+                    os.makedirs(parent_dirs)
```

Comment:
You can use os.makedir(..., exist_ok=True)

Reply:
Sure can do. Should mention that its a python 3.2+ option

Reply:
cx-freeze is only py35+

Reply:
That's good :)